### PR TITLE
docker digests as alternative names

### PR DESCRIPTION
**Pull request by TomasTomecek**
@rpitonak would you take a look when you get back?

### CODE REVIEW

**Comment by TomasTomecek** [comment]
this should be done now

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/SR2CL](https://da.gd/SR2CL)__
* Comment: __See details for logs__

**Comment by TomasTomecek** [comment]
looks like I have thngs to fix

**Comment by TomasTomecek** [comment]
I'm not capable of doing small pull requests -_-

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/KWJ49](https://da.gd/KWJ49)__
* Comment: __See details for logs__

File: conu/backend/docker/image.py [hunk 1]
```diff
@@ -265,7 +264,8 @@ def _run_container(self, run_command_instance, callback):
             container_id = fd.read()
         return container_id, response
 
-    def run_via_binary(self, run_command_instance=None, command=None, volumes=None, additional_opts=None, *args, **kwargs):
+    def run_via_binary(self, run_command_instance=None, command=None, volumes=None,
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply conu/backend/docker/image.py]: expected

File: conu/backend/docker/utils.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    #   {'HostIp': '0.0.0.0', 'HostPort': '123'},
+    #   {'HostIp': '0.0.0.0', 'HostPort': '1234'}]}
+    port_mappings = dict()
+
+    raw_port_mappings = graceful_get(inspect_data, 'HostConfig', 'PortBindings') or {}
+
+    for key, value in raw_port_mappings.items():
+        for item in value:
+            if key in port_mappings.keys():
+                if item['HostPort'] is not '':
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply conu/backend/docker/utils.py]: should this be `if item[...]:`?

[the remark this answers is not in the document]
Comment (jpopelka) [reply conu/backend/docker/utils.py]: or `if item['HostPort'] != '':`

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/xQczc](https://da.gd/xQczc)__
* Comment: __See details for logs__

**Comment by TomasTomecek** [comment]
There is still room for improvement, especially the todo from last comment.

**Comment by jpopelka** [comment]
[test]

**Comment by rpitonak** [comment]
LGTM, let's merge this and create follow-up issue for TODO comment.

**Comment by TomasTomecek** [comment]
So click the button if you are brave enough :)

**Pull request merged**